elytron-oidc-client/src/test/java/org/wildfly/extension/elytron/oidc/ExpressionsTestCase.java

```diff
@@ -35,14 +36,20 @@ public ExpressionsTestCase() {
     public void testExpressions() throws Throwable {
         if (services != null) return;
         String subsystemXml = "oidc-expressions.xml";
-        services = super.createKernelServicesBuilder(new DefaultInitializer()).setSubsystemXmlResource(subsystemXml).build();
+        services = super.createKernelServicesBuilder(new DefaultInitializer(Stability.COMMUNITY)).setSubsystemXmlResource(subsystemXml).build();
```

Comment:
You should parameterize this test such that it verifies logic for all subsystem namespace versions/stabilities.

Reply:
Do we necessarily need to parametrize this for all versions? 
We are parametrizing the subsystem parsing test cases since it was previously testing the parser for each version. However, here, we are testing expression resolution only for the latest version. I believe parametrizing it would require additional XML config files, and 3 tests instead of 1. 
What do you think?

Reply:
I see.  In this case, we would still want to parameterize this test, but limit the parameters to all *current* schema versions, e.g. ElytronOidcSubsystemSchema.CURRENT.values()